web/modules/custom/dpl_paragraphs/dpl_paragraphs.module

```diff
@@ -223,7 +223,61 @@ function dpl_paragraphs_preprocess_paragraph__material_grid_automatic(array &$va
       'title' => $material_grid_title,
       'cql' => $cql,
       'selected-amount-of-materials-for-display' => $selected_amount_of_materials_for_display,
-      'button-text' => $button_text,
+      'button-text' => t('Show all'),
```

Comment:
Please rememer to add a context to the translateable string.

Reply:
Weird, this is not there when i look. 

It should already be  ` 'button-text' => t('Show all'),`for both preprocess'es/apps